src/fixed/method.rs

```diff
@@ -50,12 +58,20 @@ impl Method for FixedMethod {
     fn finish_input_session(&mut self) {
         self.buffer.clear();
         self.typed.clear();
+        self.pending_kar = None;
     }
 
     fn backspace_event(&mut self, config: &Config) -> Suggestion {
         if !self.buffer.is_empty() {
-            // Remove the last character from buffer.
-            self.buffer.pop();
+            // TODO:
+            // - Allow to remove pending kar at the begining of a word.
```

Comment:
@NabilSnigdho can you elaborate a bit on what you exactly meant by this `TODO`? Does this mean for backspace when the buffer has `কি`, it would pop the `ক` first while `ি` will remain?

Reply:
No, actually I meant for backspace after
```
method.buffer = "".to_string();
method.process_key_value("ি", &config);
```
, it would just remove the `pending_kar`.

---

I initially tried
```
fn backspace_event(&mut self, config: &Config) -> Suggestion {
    if let Some(_) = &self.pending_kar {
        self.pending_kar = None;
        self.typed.pop();
        if self.buffer.is_empty() {
            return Suggestion::empty();
        }
        return self.create_suggestion(config);
    }
    if !self.buffer.is_empty() {
        // Remove the last character from buffer.
        self.buffer.pop();
...
```
which was not working.

I've just found out that backspace is ignored when `riti_context_ongoing_input_session` returns false. So `fn ongoing_input_session` also needs to be changed
```
fn ongoing_input_session(&self) -> bool {
    !self.buffer.is_empty()
        || match self.pending_kar {
            None => false,
            _ => true,
        }
}
```
I shall create another pull request 😔.
Sorry for this mess.

Reply:
Thanks! I also missed that, nothing to be sorry about! :smile: